UM/Settings/ContainerQuery.py

```diff
@@ -56,50 +60,50 @@ def getResult(self) -> Optional[List[Dict[str, Any]]]:
     def isIdOnly(self) -> bool:
         return len(self._kwargs) == 1 and not self._ignore_case and "id" in self._kwargs
 
-    ##  Check to see if any of the kwargs is a Dict, which is not hashable for query caching.
-    #
-    #   \return True if this query is hashable.
-    def isHashable(self) -> bool:
-        for kwarg in self._kwargs.values():
-            if isinstance(kwarg, dict):
-                return False
-        return True
-
     ##  Execute the actual query.
     #
     #   This will search the container metadata of the ContainerRegistry based
     #   on the arguments provided to this class' constructor. After it is done,
     #   the result can be retrieved with getResult().
     def execute(self, candidates: Optional[List[Any]] = None) -> None:
+        key_so_far = (self._ignore_case, )  # type: Tuple[Any, ...]
```

Comment:
This little nugget needs a bit more documentation as to what it does and how it makes things better.

Reply:
Done here: e8a1a32017e031a8cffea91dc53695a32d59942e